CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 Tags have the format: `<MediaWiki core version>-<PHP Version>-<date>-<build number>`
 
+## 1.37-7.4-20220930-fp-beta-0
+- Add option to add mediawiki patches with pacman
```

Comment:
to be boring; we should probably mention that this patch adds the actual script for unit rewriting?

Reply:
sure!